configs/clip.yml

```diff
@@ -1,22 +1,35 @@
 # Model
-feature_type: 'clip'
-model_name: 'ViT-B/32' # ViT-B/16 RN50x16 RN50x4 RN101 RN50
+feature_type: "clip"
```

Comment:
well..., i guess that's fine

Reply:
Sorry, this is autoformatter; I'll disable it